src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -493,3 +496,20 @@ void CalculatorApp::GraphingCalculator::ActiveTracing_KeyUp(Windows::UI::Core::C
         args->Handled = true;
     }
 }
+
+void GraphingCalculator::GraphSettingsButton_Click(Object ^ sender, RoutedEventArgs ^ e)
+{
+    if (m_flyoutGraphSettings == nullptr)
+    {
+        auto graphSettings = ref new GraphingSettings();
+        graphSettings->SetGrapher(this->GraphingControl);
+        m_flyoutGraphSettings = ref new Flyout();
+        m_flyoutGraphSettings->Content = graphSettings;
+        auto that(this);
+        m_flyoutGraphSettings->Closed += ref new Windows::Foundation::EventHandler<Platform::Object ^>(
+            [that](Platform::Object ^, Platform::Object ^) { that->GraphingNumPad->IsEnabled = true; });
+    }
+
+    GraphingNumPad->IsEnabled = false;
```

Comment:
Why is this needed? Seems weird that this is the only flyout that does this

Reply:
We can also use `KeyboardShortcutManager::HonorShortcuts` too but I prefer this way:
- We should probably start using KeyboardAccelerator instead of this custom implementation
-  `KeyboardShortcutManager::HonorShortcuts` prevents us to use shortcuts in the popup

Reply:
Why do we need to block shorcuts from the flyout menu? Does it cause it to close or something?

Reply:
to prevent the keys to blink when you press numbers

Reply:
@EriWong removed the blinking already and that should be in master, so this shouldn't be necessary anymore :)

Reply:
These buttons don't blink in master, no need to add this code.

Reply:
In this case you are right, this code is no more necessary! Removed!